snap/implicit_test.go

```diff
@@ -20,56 +20,9 @@
 package snap_test
 
 import (
-	"github.com/snapcore/snapd/interfaces/builtin"
-	"github.com/snapcore/snapd/release"
-	"github.com/snapcore/snapd/snap"
-
 	. "gopkg.in/check.v1"
 )
 
-type SpecialSuite struct{}
-
-var _ = Suite(&SpecialSuite{})
-
-func (s *InfoSnapYamlTestSuite) TestAddImplicitSlotsOutsideClassic(c *C) {
-	restore := release.MockOnClassic(false)
-	defer restore()
-
-	osYaml := []byte("name: ubuntu-core\ntype: os\n")
-	info, err := snap.InfoFromSnapYaml(osYaml)
-	c.Assert(err, IsNil)
-	snap.AddImplicitSlots(info)
-	c.Assert(info.Slots["network"].Interface, Equals, "network")
-	c.Assert(info.Slots["network"].Name, Equals, "network")
-	c.Assert(info.Slots["network"].Snap, Equals, info)
-	// Ensure that we have *some* implicit slots
-	c.Assert(len(info.Slots) > 10, Equals, true)
-}
-
-func (s *InfoSnapYamlTestSuite) TestAddImplicitSlotsOnClassic(c *C) {
-	restore := release.MockOnClassic(true)
-	defer restore()
-
-	osYaml := []byte("name: ubuntu-core\ntype: os\n")
-	info, err := snap.InfoFromSnapYaml(osYaml)
-	c.Assert(err, IsNil)
-	snap.AddImplicitSlots(info)
-	c.Assert(info.Slots["unity7"].Interface, Equals, "unity7")
-	c.Assert(info.Slots["unity7"].Name, Equals, "unity7")
-	c.Assert(info.Slots["unity7"].Snap, Equals, info)
-	// Ensure that we have *some* implicit slots
-	c.Assert(len(info.Slots) > 10, Equals, true)
-}
+type implicitSuite struct{}
```

Comment:
There are some things that we could test, I think. I can remove it or fill in the tests.